ibctest/docker.go

```diff
@@ -0,0 +1,74 @@
+package ibctest
+
+import (
+	"bufio"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"path/filepath"
+	"runtime"
+	"testing"
+
+	dockertypes "github.com/docker/docker/api/types"
+	"github.com/docker/docker/pkg/archive"
+	"github.com/moby/moby/client"
+	"github.com/stretchr/testify/require"
+)
+
+const RelayerImageName = "ibctestrelayer"
+
+type dockerLogLine struct {
+	Stream      string            `json:"stream"`
+	Aux         any               `json:"aux"`
+	Error       string            `json:"error"`
+	ErrorDetail dockerErrorDetail `json:"errorDetail"`
+}
+
+type dockerErrorDetail struct {
+	Message string `json:"message"`
+}
```

Comment:
Looks like these overlap with github.com/docker/docker/pkg/streamformatter and github.com/docker/docker/pkg/jsonmessage. It might make more sense to use those, but it's probably fine as-is too.